examples/exotica_examples/scripts/example_aico.py

```diff
@@ -17,7 +17,8 @@ def figureEight(t):
         problem.setRho('Frame',0.0,t)
     else:
         problem.setRho('Frame',1e5,t)
-        problem.setGoal('Frame',figureEight(t*problem.tau),t)
+        goal=figureEight(t*problem.tau)
```

Comment:
This is not PEP8 compliant - it wasn't before but in edits we should respect that.